scaleway/resource_lb.go

```diff
@@ -185,3 +206,42 @@ func resourceScalewayLbDelete(ctx context.Context, d *schema.ResourceData, meta
 
 	return nil
 }
+
+//resourceScalewayLbUpgradeZoned allow upgrade the zoned resource. Note: please create another method for future upgrades not related
+// from version 0 to 1.
+func resourceScalewayLbUpgradeIPID(ctx context.Context, rawState map[string]interface{}, meta interface{}) (map[string]interface{}, error) {
+	select {
```

Comment:
Why do we need a select here ?

Reply:
When Done is used we the the select statements. https://golang.org/pkg/context/#Context